plugins/de.cau.cs.kieler.klighd/META-INF/MANIFEST.MF

```diff
@@ -8,7 +8,8 @@ Bundle-RequiredExecutionEnvironment: JavaSE-11
 Require-Bundle: de.cau.cs.kieler.klighd.kgraph,
  de.cau.cs.kieler.klighd.krendering;visibility:=reexport,
  org.eclipse.core.runtime;bundle-version="3.6.0";visibility:=reexport,
- org.eclipse.elk.core.service;bundle-version="0.1.0",
+ org.eclipse.elk.core,
+ org.eclipse.elk.core.service,
```

Comment:
Does this change in ELK also affect the Maven dependencies in any way? In that case, the corresponding pom.xml (and pom.eclipse.xml and pom.lsp.xml, see their documentation up top) should be adapted as well.

Reply:
No it doesn't. In Maven all dependencies are transitive automatically.
And one can only prevent that by explicitly excluding certain artifacts. But as far as I can neither KLighD nor ELK excludes dependencies, does it?